openassessment/assessment/test/test_staff.py

```diff
@@ -33,7 +33,7 @@ class TestStaffAssessment(CacheResetTest):
     """
 
     STEP_REQUIREMENTS = {}
-    STEP_REQUIREMENTS_WITH_STAFF = {'staff': {'required': True}}
+    STEP_REQUIREMENTS_WITH_STAFF = {'required': True}
```

Comment:
You're right - it is related to the bug changes.

I had been carelessly double-unpacking the expected `{'staff': {'required': True}}` dictionary, first here: https://github.com/edx/edx-ora2/blob/efischer/enable_staff_grading/openassessment/workflow/models.py#L252, then again here: https://github.com/edx/edx-ora2/blob/ora-staff-grading/openassessment/assessment/api/staff.py#L57. That second location has been updated to this: https://github.com/edx/edx-ora2/blob/efischer/enable_staff_grading/openassessment/assessment/api/staff.py#L61

So this is to have the assessment level test call into its api with the actual, not the expected-but-unverified, value being passed in by the workflow model.